File: experiments/020-PythonVsJava/cells/java-gradle_1-Empty_free/2019-2c-parcial2-masked/20260921-214358-38088/output/project/src/main/java/surveycrawler/TurnCounterClockwiseCommand.java
package surveycrawler;

public class TurnCounterClockwiseCommand extends RepeatableCrawlerCommand {

    // testing

    @Override
    public boolean isFor(char aCommand) {
        return aCommand == 'g';
    }

    // executing

    @Override
    protected void executeOnce(SurveyCrawler aCrawler) {
        aCrawler.turnCounterClockwise();
    }
}

File: experiments/020-PythonVsJava/cells/python-pytest_1-Empty_free/2019-2c-parcial2-masked/20260921-202806-59410/workdir/src/surveycrawler/turn_counter_clockwise_command.py
from surveycrawler.crawler_command import CrawlerCommand


class TurnCounterClockwiseCommand(CrawlerCommand):

    # command character

    @classmethod
    def command_character(cls):
        return 'g'

    # executing

    def execute(self, a_crawler):
        a_crawler.turn_counter_clockwise()
